Invoke-Selfcheck.ps1

```diff
@@ -0,0 +1,52 @@
+Param (
+    [Parameter(Mandatory=$false)] [string] $TestenvConfFile,
+    [switch] $SkipUnit,
+    [switch] $NoStaticAnalysis
+)
+
+# NOTE TO DEVELOPERS
+# ------------------
+# The idea behind this tool is that anyone can run the basic set of tests without ANY preparation 
+# (except Pester).
+# A new developer should be able to run `.\Invoke-Selfcheck.ps1` and it should pass 100% of the time,
+# without any special requirements, like libraries, testbed machines etc.
+# Special flags may be passed to invoke more complicated tests (that have requirements), but
+# the default should require nothing.
+
+
+$nl = [Environment]::NewLine
+
+function Get-Separator {
+    $nl + ((@("=") * 80) -join "") + $nl
+}
+
+function Write-VisibleMessage {
+    param([string] $Message)
+    Write-Host "$(Get-Separator)[Selfcheck] $Message$(Get-Separator)"
+}
+
+if ($SkipUnit) {
+    Write-VisibleMessage "-SkipUnit flag set, skipping unit tests"
```

Comment:
Why did you even create a function like `Write-VisibleMessage`. What's wrong with normal `Write-Host`? Please don't write code for inserting 80 equals signs.

Reply:
I added separators because I wanted it to standout more from Pester reports.

I'm going to remove `Get-Separator` func because of this cool syntax `"=" * 80` but I think `Write-VisibleMessage` should stay. In other case, every `Write-Host` would have to add separators and `[Selfcheck]` prefix which looks ugly.